src/ServerMainComponent.cpp

```diff
@@ -175,6 +179,35 @@ isobus::VirtualTerminalBase::SupportedWideCharsErrorCode ServerMainComponent::ge
 	return isobus::VirtualTerminalBase::SupportedWideCharsErrorCode::AnyOtherError;
 }
 
+bool ServerMainComponent::keyPressed(const KeyPress &key, Component *originatingComponent)
```

Comment:
Will this repeat if held? If yes, that's probably a bug, and if no, we probably have to add handling for the still held message?

Reply:
Ah, I forgot to mention: the standard is quite foggy on this one, so I tested with a real terminal (Kverneland Tellus) and it only sends press and release events.
FYI: the alarms masks got cancelled on the press event.